velox/substrait/tests/TestUtils.cpp

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "TestUtils.h"
+#include <fstream>
+#include <sstream>
+
+void TestUtils::getMsg(
```

Comment:
TestUtils is not a very descriptive name. It looks like this class is used to read JSON from a file and parse it to Protobuf message. Would it make sense to rename to JsonToProtoConverter::readFromFile()?

Reply:
Fixed.